noggin/security/ipa_admin.py

```diff
@@ -10,6 +10,7 @@ class IPAAdmin(object):
         "user_add",
         "user_show",
         "user_mod",
+        "user_del",
```

Comment:
Where is this used? I don't think it should be necessary, noggin does not (and should not be allowed to) delete users. If it's only in the tests it should be in the `__WRAPPED_METHODS_TESTING` tuple below (it's already there).

Reply:
Probably just a case of cargo-culting. :wink:

Reply:
Removing this, and the added `pwpolicy_find` because it's not even used in testing.